docs/general/mainnet/damask-upgrade.md

```diff
@@ -0,0 +1,349 @@
+# Damask Upgrade
+
+This document provides an overview of the changes for the Damask Mainnet
+upgrade.
+
+:::caution
+
+The Damask upgrade on Mainnet will kick off around **Apr 11, 2022 at 9:00 UTC**.
```

Comment:
We should probably specify the epoch as well?

Reply:
Done.